ONSdigital/eq-author-app
# Add validation to empty routing answers

### What is the context of this PR?

Add validation to routing pages. This focuses on no answer being selected when initialising routing rule. Picking no answer causes runner to 500.

There are a series of following tickets to add validation in the same area so I've tried to find a way that will make it easy to incorporate those rules

### How to review 

- Create questionnaire with a routing rule added
- When no question is picked a warning should appear
- Delete the rule and add a new one to see if that works
